StencilSwiftKit.xcworkspace/xcshareddata/StencilSwiftKit.xcscmblueprint

```diff
@@ -0,0 +1,30 @@
+{
+  "DVTSourceControlWorkspaceBlueprintPrimaryRemoteRepositoryKey" : "804E0A642F9B0288E24EC6843BD9BD71D8BD7EA7",
```

Comment:
I never know if this file should be pushed on git, I see it being ignored by a lot of .gitignored on OSS projects and I generally gitignore it too, I think because otherwise it gets updated almost on every commit… I believe it's only useful for Xcode to have metadata about the repo, but who uses Xcode's integrated SCM tool anyway